contrib/scripts/functions.sh

```diff
@@ -27,7 +27,7 @@ function restartCluster {
     docker_compose_gopath=`pwd`/../osx-docker-gopath
 
     # FIXME: read the go version from a constant
-    docker run -it --rm \
+    docker run --rm \
```

Comment:
I removed -it. I think in a previous commit i put -it, though this isn't interactive